docs/README.md

```diff
@@ -79,7 +79,7 @@ a placeholder.
 
 ## Components
 
-### `<PhoneField />`
+### PhoneField component
```

Comment:
Since the parent title is `Components`, I think adding `component` to each subtitle is redundant.